applications/osb-portal/src/components/drawer/WorkspaceDrawer.tsx

```diff
@@ -57,22 +87,77 @@ const useStyles = makeStyles(theme => ({
   },
   content: {
     flexGrow: 1,
-    padding: theme.spacing(3),
-    transition: theme.transitions.create('margin', {
-      easing: theme.transitions.easing.sharp,
-      duration: theme.transitions.duration.leavingScreen,
-    }),
-    marginLeft: -drawerWidth,
+    padding: 0,
   },
-  contentShift: {
-    transition: theme.transitions.create('margin', {
-      easing: theme.transitions.easing.easeOut,
-      duration: theme.transitions.duration.enteringScreen,
-    }),
-    marginLeft: 0,
+  closedTextBottom: {
+    writingMode: 'vertical-lr',
+    textOrientation: 'mixed',
+    transform: 'rotate(-180deg)',
+    margin: 'auto',
+    position: 'relative',
+    bottom: 0,
+    height: '70vh',
   },
+  closedText: {
+    writingMode: 'vertical-lr',
+    textOrientation: 'mixed',
+    transform: 'rotate(-180deg)',
+    margin: 'auto',
+  },
+  rotate180: {
+    transform: 'rotate(-180deg)',
+  },
+  svgIconRight: {
+    paddingTop: 6,
+    paddingLeft: 4,
+    float: 'right',
```

Comment:
Flex justification is better than float as it plays better with container's height